karapace/kafka_rest_apis/__init__.py

```diff
@@ -400,7 +400,7 @@ async def topic_publish(self, topic: str, content_type: str, *, request):
         await self.publish(topic, None, content_type, request.content_type, request.json)
 
     @staticmethod
-    def validate_partition_id(partition_id: str, content_type: str) -> int:
+    def validate_partition_id(partition_id: str, content_type: str) -> int:  # pylint: disable=inconsistent-return-statements
```

Comment:
Note for later action: refactor `KafkaRest.not_found` and `.r` methods such that the code has `raise` statement here instead of there. Should improve readability, and avoid this pylint ignore, right?